### ISSUE TITLE
 MNIST GAN

### ISSUE THREAD

**Issue by onoriodeunu** [udacity/deep-learning-v2-pytorch#281]
Notebook: MNIST GAN
Now, you're ready to define and train a GAN of your own design. The next notebook will be all about generating new images of handwritten digits that look as if they've come from the original MNIST training data. To generate new images, a Generator must learn about the features that make up images of digits and combine those features in new ways; imagining totally new images of digits!

It's suggested that you open the notebook in a new, working tab and continue working on it as you go through the instructional videos in this tab. This way you can toggle between learning new skills and coding/applying new skills.

To open this notebook, you have two options:

Go to the next page in the classroom (recommended).
Clone the repo from Github and open the notebook MNIST_GAN_Exercise.ipynb in the gan-mnist folder. You can either download the repository with git clone https://github.com/udacity/deep-learning-v2-pytorch.git, or download it as an archive file from this link.
Instructions
Define two adversarial networks; a Discriminator and Generator
Define two loss functions that indicate how close a given image is to be classified as either real or fake
Train both networks
Visualize the loss and some sample, generated data
This is a self-assessed lab. If you need any help or want to check your answers, feel free to check out the solutions notebook in the same folder, or by clicking here.

**Comment by onoriodeunu** [comment]
Notebook: MNIST GAN
Now, you're ready to define and train a GAN of your own design. The next notebook will be all about generating new images of handwritten digits that look as if they've come from the original MNIST training data. To generate new images, a Generator must learn about the features that make up images of digits and combine those features in new ways; imagining totally new images of digits!

It's suggested that you open the notebook in a new, working tab and continue working on it as you go through the instructional videos in this tab. This way you can toggle between learning new skills and coding/applying new skills.

To open this notebook, you have two options:

Go to the next page in the classroom (recommended).
Clone the repo from Github and open the notebook MNIST_GAN_Exercise.ipynb in the gan-mnist folder. You can either download the repository with git clone https://github.com/udacity/deep-learning-v2-pytorch.git, or download it as an archive file from this link.
Instructions
Define two adversarial networks; a Discriminator and Generator
Define two loss functions that indicate how close a given image is to be classified as either real or fake
Train both networks
Visualize the loss and some sample, generated data
This is a self-assessed lab. If you need any help or want to check your answers, feel free to check out the solutions notebook in the same folder, or by clicking here.
